EXERCISE_02/app/src/main/java/com/example/exercise_02/Data.java

```diff
@@ -0,0 +1,12 @@
+package com.example.exercise_02;
+
+public class Data {
+    public int imageId;
+    public String txt;
```

Comment:
Also for properties, what does txt means? the character name? so why not calling it name?